subsys/bootloader/cmake/debug_keys.cmake

```diff
@@ -8,25 +8,42 @@ set(DEBUG_SIGN_KEY ${PROJECT_BINARY_DIR}/GENERATED_NON_SECURE_0.pem)
 set(DEBUG_PUBLIC_KEY_0 ${PROJECT_BINARY_DIR}/GENERATED_NON_SECURE_PUBLIC_0.pem)
 set(DEBUG_PUBLIC_KEY_1 ${PROJECT_BINARY_DIR}/GENERATED_NON_SECURE_PUBLIC_1.pem)
 
-set(cmd
+set(privcmd
   ${PYTHON_EXECUTABLE}
-  ${NRF_BOOTLOADER_SCRIPTS}/keygen.py
+  ${NRF_BOOTLOADER_SCRIPTS}/keygen.py --private
   )
+#set(privcmd
+#  openssl ecparam -name prime256v1 -genkey -noout
+#  )
+#set(privcmd
+#  openssl genpkey -algorithm rsa -pkeyopt rsa_keygen_bits:2048
+#  )
+
+set(pubcmd
+  ${PYTHON_EXECUTABLE}
+  ${NRF_BOOTLOADER_SCRIPTS}/keygen.py --public
+  )
+#set(pubcmd
+#  ${privcmd} | openssl ec -pubout
+#  )
+#set(pubcmd
+#  ${privcmd} | openssl rsa -pubout
+#  )
 
 add_custom_command(
   OUTPUT
   ${DEBUG_SIGN_KEY}
   ${DEBUG_PUBLIC_KEY_0}
   ${DEBUG_PUBLIC_KEY_1}
   COMMAND
-  ${cmd}
-  --output-private ${DEBUG_SIGN_KEY}
+  ${privcmd}
+  -out ${DEBUG_SIGN_KEY}
```

Comment:
Would it be better to use `--out` as we do with all other multi-character arguments?

Reply:
It's from experimenting with the openssl CLI. I'll change it back for consistency.

Reply:
Done